enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -8,12 +8,8 @@
 
 {% block contents %}
   <div class="enterprise-container">
-    <div class="row no-gutters">
-      <div class="col-3">
-        {% include "enterprise/_enterprise_customer_sidebar.html" %}
-      </div>
-      <div class="col-7 border-left">
-
+    <div class="row">
+      <div class="col-8">
```

Comment:
```suggestion
      <div class="col-8" style="margin: auto;">
```
What do you think of adding this style to center the container?